src/Python/omop_table_examples.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import requests
+import json
+from IPython.display import display
+
+from src.Python.metadata_examples import examples
+
+
+def omop_domain(field):
+    concept_id = int(examples[examples['field_id'] == field]['target_concept_id'].unique())
+    response = requests.get(f'https://athena.ohdsi.org/api/v1/concepts/{concept_id}')
+    concept = json.loads(response.text)
+    domain = concept['domainId']
+    return domain
+
+
+def example_omop_table(field, instance, *value):
+    domain = omop_domain(field)
+    if value:
+        row = examples[(examples['field_id'] == field) & (examples['value'] == value)]
+    else:
+        row = examples[examples['field_id'] == field]
+
+    table = pd.Series({'person_id': 'get(eid)',
```

Comment:
This does not take the mapping approach into account currently. 
e.g. for a numeric approach you would not have a value_as_concept_id.